spec/index.html

```diff
@@ -1639,6 +1698,28 @@ <h2>Entailment rules (Informative)</h2>
 
   <p>As noted earlier, detecting datatype entailment for larger sets of datatype IRIs
     requires attention to idiosyncratic properties of the particular datatypes.</p>
+    
+  <p>
+  The complete entailment pattern for generalized RDF with [=symmetric RDF triple=]s, considering that, according to the semantics, the denotation of triple terms should be of type <code>rdfs:Proposition</code>, is the following:
```

Comment:
Line length again. `=]s` should not exist. Any terms which are currently treated this way should get the plural added to the synonyms of the singular, and the `s=]` should become `s=]`. (Similar treatment also applies to other plural forms.)
```suggestion
  The complete entailment pattern for generalized RDF with [=symmetric RDF triples=],
  considering that, according to the semantics, the denotation of triple terms should
  be of type <code>rdfs:Proposition</code>, is the following:
```